testpilot/frontend/static-src/styles/_hidpi-mixin.scss

```diff
@@ -0,0 +1,20 @@
+// This is a hack because node-sass does not support
```

Comment:
Can you add a "TODO:" to the front of this so we can search it easier.
Also, replace "node-sass" with "sass-lint"

Relevant bug `https://github.com/sasstools/sass-lint/issues/70`

:heart: